src/Core/Domain/Order/Exception/InvalidRefundException.php

```diff
@@ -31,7 +31,7 @@
 /**
  * Throw new when a partial refund's is asked without a specified quantity
  */
-class InvalidRefundQuantityException extends OrderException
+class InvalidRefundException extends OrderException
```

Comment:
Do you consider that having a different exception for quantity and amount was overkill? (or is there another reason for this slight refactor ?)

Reply:
Well, fo two scenarii I didn't mind But then I added more specific use case codes, so having 5 different exceptions related to a refund validation seemed overkill indeed So I preferred to mix them all

Reply:
alright it makes sense 👍